docs/content/Box.md

```diff
@@ -2,17 +2,24 @@
 title: Box
 ---
 
-
-The Box component serves as a wrapper component for most layout related needs. Use Box to set values such as `display`,  `width`, `height`, and more. See the LAYOUT section of our [System Props](/system-props) documentation for the full list of available props. In practice, this component is used frequently as a wrapper around other components to achieve Box Model related styling.
+The Box component serves as a wrapper component for most layout related needs. Use Box to set values such as `display`, `width`, `height`, and more. See the LAYOUT section of our [System Props](/system-props) documentation for the full list of available props. In practice, this component is used frequently as a wrapper around other components to achieve Box Model related styling.
 
 ## Default example
 
 ```jsx live live
 <Box>
- Box can be used to create both <Box as="span" bg="green.1">inline</Box> and
- <Box bg="blue.1">block-level elements,</Box>
- <Box bg="purple.1" width={[1, 1, 1/2]}>elements with fixed or responsive width and height,</Box>
- <Box bg="yellow.0" p={4} mt={2}>and more!</Box>
+  Box can be used to create both{' '}
+  <Box as="span" bg="bg.success">
+    inline
+  </Box>{' '}
+  and
+  <Box bg="bg.danger">block-level elements,</Box>
+  <Box bg="bg.warning" width={[1, 1, 1 / 2]}>
+    elements with fixed or responsive width and height,
+  </Box>
+  <Box bg="bg.info" p={4} mt={2}>
+    and more!
+  </Box>
```

Comment:
Some of these functional color choices I was trying out end up being very light shades - we could maybe consider switching to `[success/danger/info]Inverse`?

<img width="970" alt="Screen Shot 2021-03-05 at 11 08 32 AM" src="https://user-images.githubusercontent.com/16739070/110149050-30650480-7da3-11eb-89d6-1682b8a0b7c2.png">
<img width="765" alt="Screen Shot 2021-03-05 at 11 08 43 AM" src="https://user-images.githubusercontent.com/16739070/110149063-335ff500-7da3-11eb-9e33-10829e086c78.png">

Reply:
Agreed. I think using the `*Inverse` colors for Box, Grid, and Flex examples is fine. We'd probably need to pair that with `color="text.inverse"` as well.